src/fabric/src/fabric_util.erl

```diff
@@ -105,7 +105,7 @@ get_db(DbName, Options) ->
     Nodes = [node()|erlang:nodes()],
     Live = [S || #shard{node = N} = S <- Shards, lists:member(N, Nodes)],
     Factor = list_to_integer(config:get("fabric", "shard_timeout_factor", "2")),
-    get_shard(Live, [{create_if_missing, true} | Options], 100, Factor).
+    get_shard(Live, Options, 100, Factor).
```

Comment:
One minor subtlety here is when this code gets deployed, and there is mixed cluster with both old and new nodes, the new nodes calling `fabric_rpc:open_shard/2` on old nodes won't have the `{create_if_missing, true}` flag. It's probably fine though since if a shard open fails, we'll just jump over it and try another one.

Reply:
Yeah... on a positive note it'll just keep retrying until it works. I needed to remove the `create_if_missing=true` option here, because the whole source of this bug is that without the proper db doc config properties, it's not valid to request db creation, and given how often this `get_db` codepath is called, I wanted focus on making the normal case fast and postpone looking up the dbs db docs until absolutely needed.

Reply:
Yap that makes sense. It just may be something to keep an eye on during deployment but I don't think it should be a problem in general.